tools/requirements.txt

```diff
@@ -0,0 +1,14 @@
+boto3==1.16.24
```

Comment:
On a second thought, I don't really understand why we need this unversioned file. It seems to only be used when there isn't a `requirements-freeze.txt` file present.
But this shouldn't be the case once we submit them to Github, right?

Reply:
You are right, requirements.txt won't be used in future. It's just the **base** list of packages we need for the container.  The actual list of pip vended freeze might be different after installations.

I can a separate PR to get rid of ```requirements.txt``` with some code cleanup. But as long as we refer in the code we should keep that file.

Reply:
agree with you. Maybe it makes sense to keep `requirements.txt`. It is similar to a `Cargo.toml` file, while the freezed variant is similar to `Cargo.lock`